StandardsRegExt.tex

```diff
@@ -76,7 +75,7 @@ \section*{Acknowledgements}
 Information Technology Research Program under Cooperative Agreement
 AST0122449 with The Johns Hopkins University, from the
 UK Particle Physics and Astronomy Research Council
-(PPARC)\footnote{\url{http://www.pparc.ac.uk/}}, and from the
+(PPARC)\footnote{\url{http://www.pparc.ac.uk/}\todo[inline]{BROKEN LINK}}, and from the
```

Comment:
Let's just drop the link...

Reply:
Link removed in https://github.com/ivoa-std/StandardsRegExt/pull/7/commits/86e48830fb87778927a8e2e65498a13015333c9b